spec/src/main/asciidoc/chapters/introduction/project_team.adoc

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2022 Contributors to the Eclipse Foundation
```

Comment:
If the project is established this year, should (c) not refer to the past so long ago?

Reply:
Nope, copy/paste issue.

Now fixed.
thanks.